scripts/pre_commit_linter.py

```diff
@@ -17,8 +17,9 @@
 """Pre-commit script for Oppia.
 
 This script uses the JSCS node module to lint JavaScript code, and prints a
-list of lint errors to the terminal. If the --autofix flag is passed, it will
-also attempt to automatically fix these errors.
+list of lint errors to the terminal. If the directory path is passed,
+it will lint all JavaScript files in that directory; otherwise,
+it will only lint files that have been touched in this commit.
```

Comment:
Below this, can you give some examples of usage (with and without args), similar to what's done at the top of scripts/test.sh?